CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # marshmallow_dataclass change log
 
+## v8.0.0
+ - Added support for validator stacking. This breaks backwards compatability. See https://github.com/lovasoa/marshmallow_dataclass/issues/91.
+    ### What this means:
+    ```python
+        CustomType = NewType("CustomType", str, validate=marshmallow.validate.bla())
```

Comment:
Nitpick: can we use an actual validator instead of bla ? For instance : 

```suggestion
        CustomType = NewType("CustomType", str, validate=marshmallow.validate.Length(min=3))
```

Reply:
I made the change myself. I'm going to merge  this